lib/apple_formatter.rb

```diff
@@ -40,9 +40,9 @@ def self.write_strings_file(terms, file, substitutions: nil, print_date: false,
           out_lines += ['', '/'*80, "// MARK: #{mark}"]
         end
 
-        # If plural, use the text for the "one" (singular) entry
+        # If definition is a hash, use the text for "one" if available (singular in languages using plurals), otherwise the first entry
```

Comment:
```suggestion
        # If definition is a Hash, use the text for "one" if available (singular in languages using plurals)
        # otherwise (e.g. asian language where only key in hash will be "other", not "one"), then use the first entry
```